dhall-openapi/openapi-to-dhall/Main.hs

```diff
@@ -66,7 +66,7 @@ writeDhall path expr = do
   putStrLn $ "Writing file '" <> path <> "'"
   Text.writeFile path $ pretty expr <> "\n"
 
-  let characterSet = Dhall.Pretty.ASCII
+  let chosenCharacterSet = Just Dhall.Pretty.ASCII
```

Comment:
I'm not entirely sure why `openapi-to-dhall` was defaulting to `ASCII` (and not `Unicode` like `dhall format`). Now that we have the option to infer it from the input, should we do that?

Reply:
@basile-henry: The only reason it was defaulting to ASCII is because it was factored out from `dhall-kubernetes`, which was using ASCII.  However, I think it's fine to change the behavior to infer it from the input